### ISSUE TITLE
JBL Xtreme 2 DFU file

### ISSUE THREAD

**Issue by mps2112** [M4lw4reT3sts/scripts-cheats#1]
Hello, I was wondering if you have the firmware downgrade file for Xtreme 2? I got mine yesterday and it is missing the lfm. Thanks.

P.S. I saw your "issue" on a post and you closed it.

**Comment by M4lw4reT3sts** [comment]
> Hello, I was wondering if you have the firmware downgrade file for Xtreme 2? I got mine yesterday and it is missing the lfm. Thanks.
> 
> P.S. I saw your "issue" on a post and you closed it.

I had received 3 dfu files but they don't work on the new xtreme 2 model

**Comment by M4lw4reT3sts** [comment]
[1.6 + 1.9 + 2.0.zip](https://github.com/M4lw4reT3sts/scripts-cheats/files/6771653/1.6.%2B.1.9.%2B.2.0.zip)

**Comment by mps2112** [comment]
So you did not downgrade?

**Comment by M4lw4reT3sts** [comment]
> So you did not downgrade?

Nope my version is GG and not PL so I can't use these

**Comment by mps2112** [comment]
Oh ok, mine is xtreme 2Q for some reason.

**Comment by AuztiinAnonX** [comment]
I have the older chip CSR firmware files v0.7.0, v1.1.0, v1.3.3, v1.4.2, v1.6.0, v1.9.0, v2.0.0, v2.3.0, v2.7.0
, v8.8.8 which are .dfu files to be used with ConnectPlus 1.1.3 or 1.2.0 prerelease. Plus I have the newer model chip QCC files v2.6.0, v2.7.0 and a few different v8.8.8 test files which are .bin files to be used with ConnectPlus 1.1.3 since the updated prerelease has everything but .dfu files disabled for flashing. 

https://github.com/pembem22/jbl-firmware

**Comment by Qltxi** [comment]
> I have the older chip CSR firmware files v0.7.0, v1.1.0, v1.3.3, v1.4.2, v1.6.0, v1.9.0, v2.0.0, v2.3.0, v2.7.0 , v8.8.8 which are .dfu files to be used with ConnectPlus 1.1.3 or 1.2.0 prerelease. Plus I have the newer model chip QCC files v2.6.0, v2.7.0 and a few different v8.8.8 test files which are .bin files to be used with ConnectPlus 1.1.3 since the updated prerelease has everything but .dfu files disabled for flashing.
> 
> https://github.com/pembem22/jbl-firmware

I don't think you can downgrade a QCC speaker. You were just very unlucky to have received a GT.
